cli/lib/kontena/cli/plugins/uninstall_command.rb

```diff
@@ -6,9 +6,11 @@ class UninstallCommand < Kontena::Command
     include Kontena::Cli::Common
 
     parameter 'NAME', 'Plugin name'
+
     option "--force", :flag, "Force remove", default: false, attribute_name: :forced
 
     def execute
+      exit_with_error "Plugin #{pastel.cyan(name)} is not installed" unless Kontena::PluginManager.instance.installed(name) && !ENV['NO_PLUGINS']
```

Comment:
Makes sense. The triple-negation on the `unless !ENV['NO_PLUGINS']` is just super confusing, but I trust this to do the right thing based on testing without trying to understand it :P